src/controller/shop.controller.ts

```diff
@@ -71,6 +71,14 @@ export default class ShopController extends BaseController {
     this.success(res, '--shop/deleted', 'shop deleted', 200, null);
   }
 
+  async getAllShops(req: Request, res: Response) {
+    const shops = await prisma.shop.findMany();
+    if (shops.length > 0) {
```

Comment:
secondly, you dont need to check if it empty or not, if the shop is empty, an empty array would be returned and the frontend devs can decide to show a message based on that.